src/site/content/en/blog/houdini-how/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 'Cross-browser paint worklets and Houdini.how'
+subhead: CSS Houdini worklets and resources are just a few clicks away.
+authors:
+  - una
+date: 2020-12-10
+hero: hero.jpg
+alt: A sparkler.
+description: Learn how to implement cross-browser Houdini Paint API's and explore a world of Houdini worklets with Houdini.how.
+tags:
+  - blog
+  - css
+  - houdini
+---
+
+CSS [Houdini](https://developer.mozilla.org/en-US/docs/Web/Houdini) is an umbrella term that describes a series of low-level browser APIs that give developers much more control and power over the styles they write. 
+
+<figure class="w-figure">
+  <img src="./houdini.jpg" alt="Houdini layer">
+</figure>
+
+Houdini enables more semantic CSS with the [Typed Object Model](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Typed_OM_API/Guide). Developers can define advanced CSS custom properties with syntax, default values, and inheritance through the [Properties and Values API](https://web.dev/at-property/).
+
+It also introduces paint, layout, and animation [worklets](https://developers.google.com/web/updates/2018/10/animation-worklet), which open up a world of possibilities, by making it easier for authors to hook into the styling and layout process of the browser's rendering engine.
+
+## Understanding Houdini worklets
+
+Houdini worklets are browser instructions that run off the main thread and can be called when needed. Worklets enable you to write modular CSS to accomplish specific tasks, and require a single line of JavaScript to import and register. Much like service workers for CSS style, Houdini worklets are registered to your application, and once registered can be used in your CSS by name.
+
+Write worklet file 
+Register worklet module (`CSS.paintWorklet.addModule(workletURL)`)
+Use worklet (`background: paint(confetti)`)
+
+## Implementing your own features with the CSS Painting API
+
+The [CSS Painting API](https://drafts.css-houdini.org/css-paint-api/) is an example of such a worklet (the Paint worklet), and enables developers to define canvas-like custom painting functions that can be used directly in CSS as backgrounds, borders, masks, and more. There is a whole world of possibilities for how you can use CSS Paint in your own user interfaces.
+
+For example, instead of waiting for a browser to implement an angled borders feature, you can write your own Paint worklet, or use an existing published worklet. Then, rather than using border-radius apply this worklet to borders and clipping.
+
+<figure class="w-figure">
+  <img src="angled-demo.gif" alt="Demo with angled corners and material design buttons.">
```

Comment:
Convert to MP4 please

https://web.dev/handbook/markup-media/#video-hosted-on-web.dev